platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,42 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    result="$(need_cmd apt-get)"
+    if [ $? != 0 ] ; then
+        wrap_debug "apt-get package manager is not present."
+        result="$(need_cmd dpkg)"
+        if [ $? != 0 ]; then
+            wrap_debug "dpkg package manager is not present."
+            result="$(need_cmd yum)"
+            if [ $? != 0 ]; then
+                wrap_debug "yum package manager is not present."
+                result="$(need_cmd rpm)"
+                if [ $? != 0 ]; then
+                    wrap_debug "rpm package manager is not present."
+                    not_found=1
+                else
+                    wrap_debug "rpm package manager is present."
+                fi
+            else
+                wrap_debug "yum package manager is present."
```

Comment:
I thought about this, we do not publish to yum repository and hence it is not a supported package manager, we can remove it . Right @damonbarry ?

Reply:
We don't currently publish RPMs, but we do make them available on GitHub, and very soon we'll have Tier 1 support for RHEL, which uses dnf (a high-level package manager on top of rpm). So I think it's ok to keep this.

Reply:
Is dnf similar to yum? Should we be checking for that as well?

Reply:
> Is dnf similar to yum?

Yes dnf and yum are similar, both high-level package managers on top of rpm.

> Should we be checking for that as well?

Not sure. If we check for dpkg & rpm, do we even need to check for apt, yum, or dnf?

Reply:
We can check for high-level packages in priority order like apt-get, then `dpkg`, then` rpm`..` yum`. So when we hit a stage where we find none, we throw the warning. `yum` resolves dependencies unlike `rpm`. I can change the order to `yum` first but since we do not publish to `yum` repo I have kept it last in the list